src/test/java/io/strimzi/kafka/quotas/StaticQuotaCallbackTest.java

```diff
@@ -198,7 +198,7 @@ void excludedPrincipal() {
         KafkaPrincipal foo = new KafkaPrincipal(KafkaPrincipal.USER_TYPE, "foo");
 
         target.configure(Map.of(
-                StaticQuotaConfig.EXCLUDED_PRINCIPAL_NAME_LIST_PROP, "foo,bar",
+                StaticQuotaConfig.EXCLUDED_PRINCIPAL_NAME_LIST_PROP, "User:foo;User:bar;User:CN=my-cluster,O=io.strimzi;arnost",
```

Comment:
Should there be `User:`?

```suggestion
                StaticQuotaConfig.EXCLUDED_PRINCIPAL_NAME_LIST_PROP, "User:foo;User:bar;User:CN=my-cluster,O=io.strimzi;User:arnost",
```

Reply:
I wanted to check that it will be skipped in case that it is not prefixed (so it will not fail in this case). But I can remove it from there if it doesn't make sense.

Reply:
Why do we want to skip it and not throw an error?

PS: You should also update the PR description.

Reply:
Yeah good point. I thought that we can simply ignore it. I will throw an exception then. Thanks :) 

(and update the PR description)

Reply:
Updated